client/reader/site-subscriptions-manager/site-subscriptions-manager.tsx

```diff
@@ -49,7 +52,11 @@ const SiteSubscriptionsManager = () => {
 				<HStack className="site-subscriptions-manager__header-h-stack">
 					<FormattedHeader
 						headerText={ translate( 'Manage subscribed sites' ) }
-						subHeaderText={ translate( 'Manage your newsletter and blog subscriptions.' ) }
+						subHeaderText={
+							hasTranslation( 'Manage your site, RSS, and newsletter subscriptions.' )
```

Comment:
We could consider adding a check for the locale - and if it starts with `en`, the new translation could be already displayed. Example: https://github.com/Automattic/wp-calypso/commit/aee8a55f4cbf93fc249ff947d873c3d0de93ef46.